src/collector.js

```diff
@@ -76,6 +77,19 @@ export default class Collector {
     return false;
   }
 
+  recordScannedFile(filename, scanner) {
+    // TODO: Add some code that verifies and normalizes `filename`
+    // to better avoid duplicates.
+    // See https://github.com/mozilla/addons-linter/issues/1310
+    if (filename in this.scannedFiles) {
```

Comment:
ah, I think I read about this. It doesn't seem to be a performance issue though mostly a potential for false-positives. But given that our filenames always have a `.css` or other extensions I doubt that we'll run into that.

Anything else I'm missing?